skyportal/models/observation_plan.py

```diff
@@ -58,6 +58,11 @@ class DefaultObservationPlanRequest(Base):
         doc="Content of the default observation plan request.",
     )
 
+    filters = sa.Column(
+        psql.JSONB,
+        doc="Filters of the default observation plan request.",
+    )
```

Comment:
what kind of filters are we talking about here?

Reply:
Adding some extra text here to be more explicit.